openshift/system/Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV RUBY_MAJOR_VERSION=2 \
     NODE_ENV=production
 ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
 ADD . ./
+ADD config/examples/*.yml ./config/
```

Comment:
But these files are already in the image, I wonder if this is better or it is better to later do `cp config/examples/* config/`

But should be good either way.

Pls don't forget to backport this change in the production Dockerfile also.

Reply:
Yeah, we could do that too. Let me take a look.

Reply:
Like this @akostadinov ?
https://github.com/3scale/porta/pull/3766/commits/d941b5afe0ec81eabfefa96876df47a541f868ad

Reply:
Why is `cp` better than `ADD`?

Reply:
I am not sure :grimacing: But in this case I think the benefit is that we create just one intermediate image instead of two.

Reply:
ok